06 Live Trading/07 Live Reconciliation/00 Introduction.html

```diff
@@ -0,0 +1,107 @@
+</p>
+
+<h4>Reality Modeling Error</h4>
+<p>
+  We provide models for predicting fees, slippage, and fills. These model predictions may not always match the fees 
+  that a live algorithm actually faces though. Users can edit their slippage models to better reflect their specific 
+  assets. Refer to our 
+  <a href="https://www.quantconnect.com/docs/algorithm-reference/reality-modelling">documentation</a> for assistance.
+</p>
+
+<h4>Market Impact</h4>
+<p>
+  LEAN does not currently model market impact. Thus, fill prices can be better during a backtest than in live trading. 
+  Users can follow the instructions in our 
+  <a href="https://www.quantconnect.com/docs/algorithm-reference/reality-modelling#Reality-Modelling-Fill-Models">documentation</a> 
+  to implement a custom fill model which incorporates market   impact.
+</p>
+
+<h4>Stale Fills</h4>
+<p>
+  Backtest orders that are submitted on stale data may have simulated fill prices that are different from the real 
+  market price. This commonly happens when creating a 
+  <a href="https://www.quantconnect.com/docs/algorithm-reference/scheduled-events">scheduled event</a> with an 
+  incompatible data resolution. To avoid this, we recommend using the highest resolution data as much as possible such 
+  as minute data, even if you only need hourly bars. This will give you an up to date price feed.
+</p>
+
+<h4>Different Backtest Parameters</h4>
+<p>
+  If the live deployment and backtest aren't started on the same date, cash balance, and holdings, trades and 
+  performance may differ. To resolve this, ensure the backtest parameters are the same as the live deployment.
+</p>
+
+<h4>Data Adjustments</h4>
+<p>
+  Sometimes data providers fix a bug that surfaced in their live data feed. Differences in performance can occur if 
+  the data is corrected after the live algorithm views the original data but before the backtest algorithm views it. 
+  For reference, all of our reported data issues are displayed 
+  <a href="https://www.quantconnect.com/data/issues/open">here</a>. To fix this, closely review your order fill prices 
+  to see if there were trade prices which seem unrealistic. This can be a signal of a live feed data error. It is rare
+  to see an error in the live price; but it is common to miss a split or corporate event in live trading for smaller 
+  companies. There are unfortunately no professional data feeds for corporate events.
+</p>
+
+<h4>Discrete Time Steps</h4>
+<p>
+  In a backtest, the <a href="https://www.quantconnect.com/docs/key-concepts/understanding-time">Time Frontier</a> 
+  moves in precise steps every time. Live data time moves as a stream not a set of discrete steps. Therefore, if an 
+  algorithm is checking for a specific time without using a tolerance value (i.e. time is 9:30:15am) , the logic may 
+  not work the same when switching to live trading. As a remedy, we suggest our users use a time range (time > 9.30.10 
+  AND time < 9.30.20) or use a 
```

Comment:
< and > characters will need to use HTML entities "&gt;" "&lt;" or it'll look like a HTML code item.